Emby.Server.Implementations/Services/ServiceController.cs

```diff
@@ -21,6 +34,13 @@ public void Init(HttpListenerHost appHost, IEnumerable<Type> serviceTypes)
 
         public void RegisterService(HttpListenerHost appHost, Type serviceType)
         {
+            // Make sure the provided type implements IService
+            if (!typeof(IService).IsAssignableFrom(serviceType))
```

Comment:
Why not
```suggestion
            if (!(serviceType is IService))
```
?

Reply:
`serviceType` is a `Type` variable, not an `IService`

Reply:
Right, my bad